at-hoangtran/Week3/Day5/Exercise2-Day5.rb

```diff
@@ -0,0 +1,59 @@
+class Dictionary
+  attr_accessor :dictionary, :vocabulary
+  def initialize
+    @dictionary = []
+    @vocabulary = []
+  end
+  
+  def add(vocab, desc)
+    dict = {}
+    unless @vocabulary.include? (vocab)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/074932088ed5cad5e69983d076619cff8875320e/at-hoangtran/Week3/Day5/Exercise2-Day5.rb#L10"></span>Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/rubocop-hq/ruby-style-guide#no-nested-conditionals)</td>
    </tr>
  </tbody>
</table>